pkgs/by-name/gr/grafana-to-ntfy/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
```

Comment:
`pkgs` should not be imported in this manner. It makes it a lot harder to deal with `.override`. Could you replace `pkgs` with `pkg-config`, `openssl`, etc?